x-pack/plugins/code/public/components/help_menu/help_menu.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import chrome from 'ui/chrome'
+import { documentationLinks } from '../../lib/documentation_links';
+import { EuiButton, EuiHorizontalRule, EuiText, EuiSpacer, EuiPortal } from '@elastic/eui';
+
+export class HelpMenu extends React.PureComponent {
+  state = { isFlyoutVisible: false };
+
+  showFlyout = () => {
+    this.setState({ isFlyoutVisible: true });
+  };
+
+  hideFlyout = () => {
+    this.setState({ isFlyoutVisible: false });
+  };
+
+  public render() {
+    return (
+      <React.Fragment>
+        <EuiHorizontalRule margin="none" />
+        <EuiSpacer />
+        <EuiText size="s">
+          <p>For Code specific information</p>
+        </EuiText>
+        <EuiSpacer />
+        <EuiButton fill iconType="popout" href={chrome.addBasePath('/app/code#/setup-guide')} >Setup Guide</EuiButton>
```

Comment:
is it possible to pass router information so we can use link to?

Reply:
this component is mounted in a way that is not wrapped by the Router our main app uses, so it won't be accessible to the state of the main app. but if this is a one-time use component, we can do some manual work to parse the browser URL ourselves.